src/routes/settings/assistants/[assistantId]/+page.ts

```diff
@@ -1,14 +0,0 @@
-import { base } from "$app/paths";
```

Comment:
I deleted `+page.ts` that uses `const data = await parent();`, which I believe was causing the freezing.

As a replacement, I've put the `load` in `+page.server.ts` that uses mongodb rather than `const data = await parent();` to get the assistant info.

Was there a specific reason to prefer using `load` in `+page.ts` over  `+page.server.ts`? (cc: @nsarrazin )

Reply:
if it looks good, I can also update the model settings equavalent https://github.com/huggingface/chat-ui/blob/fix_settings_freeze/src/routes/settings/%5B...model%5D/%2Bpage.ts

Reply:
I think the reasoning was to filter the correct assistant from parent data to reduce the number of DB calls. 

Obviously that doesn't seem to have helped with performance :laughing: But this looks good!

Reply:
> if this PR looks good, I can also update the model settings equavalent https://github.com/huggingface/chat-ui/blob/fix_settings_freeze/src/routes/settings/%5B...model%5D/%2Bpage.ts

actually, this is not possible atm because models are not in db. They are rather in memory. Therefore, the current approach is sufficient